subsys/spm/Kconfig

```diff
@@ -52,9 +52,19 @@ config PM_PARTITION_SIZE_SPM_SRAM
 config SPM_BOOT_SILENTLY
 	bool "Boot silently"
 	default n
+
+config SPM_SHARE_CONSOLE_UART
+	bool "Enable sharing console between domains"
+	help
+	  When enabled, UARTE0 can be used for printing from secure domain
+	  after jumping to the non-secure application, e.g. from exception
+	  handler. If enabled, zephyr console must be changed do spm_uart in
+	  the device tree configuration.
```

Comment:
it states `If enabled, zephyr console must be changed do spm_uart in the device tree configuration.` I didn't want to explicitly mention overlay here because overlay is in the sample and here we are in subsys.

Reply:
understood.